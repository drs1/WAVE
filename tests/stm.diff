*** ../tests/stm.out	2014-12-03 04:34:42.000000000 -0500
--- ./wave2.out	2014-12-03 04:34:46.000000000 -0500
***************
*** 1,8 ****
  100
  98
  102
- 0
- 100
- 318767205
- 102
- 0
--- 1,3 ----
