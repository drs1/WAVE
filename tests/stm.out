100
98
102
0
100
318767205
102
0
